libclc/opencl/include/clc/opencl/synchronization/utils.h

```diff
@@ -21,4 +22,14 @@ _CLC_INLINE int getCLCMemoryScope(cl_mem_fence_flags flag) {
   return memory_scope;
 }
 
+_CLC_INLINE MemorySemantics getCLCMemorySemantics(cl_mem_fence_flags flag) {
```

Comment:
Doesn't follow clc naming conventions (enum name too), but that already appears to be an issue in this file

Reply:
> Doesn't follow clc naming conventions (enum name too), but that already appears to be an issue in this file

Done in https://github.com/llvm/llvm-project/pull/152275/commits/02e3047ae448c3925d1814cdbfbb148613ee0fb5 
* Added `__CLC_` prefix to `MemorySemantics` and its enum values
* Added `__opencl_` prefix to functions in libclc/opencl/include/clc/opencl/synchronization/utils.h